pom.xml

```diff
@@ -117,7 +117,7 @@
             <dependency>
                 <groupId>com.autonomy.aci.client</groupId>
                 <artifactId>autonomy-aci-client-api</artifactId>
-                <version>4.1.3</version>
+                <version>12.4.0</version>
```

Comment:
I struggle to believe this change is correct because I don't see a `com.autonomy.aci.client:autonomy-aci-client-api:12.4.0` in [Maven Central](https://repo1.maven.org/maven2/) (or in Artifactory).

Reply:
I can't find it there either. I also tried to update [util-aci](https://github.houston.softwaregrp.net/caf/util-aci/pull/6) but it throws an error:
```
Could not find artifact com.autonomy.aci.client:autonomy-aci-client-api:jar:12.4.0 in gsts-company-central (https://svsartifactory.swinfra.net/artifactory/company-central)
```
I am trying to find out what differs between these two.